app/bot.py

```diff
@@ -17,13 +17,19 @@
 logger = logging.getLogger()
 
 bot_name = 'RowenWitt' # Need bot name
+bot_id = 1335727237400694784
+welcome_message_id = 1
+dm_link = f'https://twitter.com/messages/compose?recipient_id={bot_id}&welcome_message_id={welcome_message_id}'
 
 conversation_tree = {
 	1:"Hi, do you have more information about the location of this incident?",
 	2:"What is the location where this incident took place?",
 	3:"Thanks! You're helping (align incentives)!",
 	4:"Thanks anyway!",
-	5:"Please fill out this form "
+	5:"Please fill out this form ",
+	10:'Click button below to start conversation ',
+	11:'Please fill out this form ',
+	12:'Thanks anyway!'
 }
 
```

Comment:
Options 4 and 12 and 5 and 11 are the same responses.  Not sure if this was intentional or just needs cleaned up.

Reply:
Intentional for now, it is just a placeholder for the eventual scripts database.